README.md

```diff
@@ -1,70 +1,229 @@
+## 🚀 Live Demo <a name="live-demo"></a>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+- [Live Demo Link](https://southamerica-countries.netlify.app/)
 
-### Deployment
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+<!-- GETTING STARTED -->
 
-### `npm run build` fails to minify
+## 💻 Getting Started <a name="getting-started"></a>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+To have installed [node.js](https://nodejs.org/en/download/) and [Git](https://git-scm.com/downloads)
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+First enter the folder you want to save the project.
+```sh
+  cd my-folder
+```
+After moving into the directory you want to save the project you could clone it using the terminal.
+
+```sh
+  git clone git@github.com:d4nQw3rty/capstone3.git
+```
+
+
+### Install
+
+Install this project with:
+
+```sh
+  cd capstone3
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm start
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  npm run test
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Danys Linares**
+
+- GitHub: [@github](https://github.com/d4nQw3rty)
+- Twitter: [@twitter](https://twitter.com/Danys_Linares)
+- LinkedIn: [LinkedIn](www.linkedin.com/in/danys-linares-6a328b238)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Improve styling of the page]**
+- [ ] **[Add more information from the API]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/d4nQw3rty/capstone3/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project leave a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+We would like to thank Microverse for letting us be part of this amazing program.
+Thanks to Nelson Sakwa for his Design.
```

Comment:
- [ ] great work. please add the [link ](https://www.behance.net/sakwadesignstudio)to the design in the name(Nelson Sakwa) . this will help take a user directly to the original design.

Reply:
well resolved! ✔️